src/Core/Config.cpp

```diff
@@ -160,6 +160,12 @@ bool Config::IsPeerBlocked (const std::string peer) {
 	return false;
 }
 
+bool Config::IsPeerPreferred (const std::string peer) {
```

Comment:
std::string => std::string&

nit: Unnecessary space between `IsPeerPreferred` and `(`

Reply:
Better yet, take in a `const IPAddress&` instead, and store `IPAddress`s when parsing the config.